PotLock/activity-dao
# Optimization

## Summary
Changes made-
Next/font utilized for faster load time. No more layout shifts based on load and FOUT (flash of unstyled text)
Took out loading animation, currently shows return <div>loading</div> to minimize any more loading time
guild logo added to footer linking to potlock guild
centered login button with current header on mobile
fixed some margins with text on feeds
added routes to tabs on feeds, go from feeds => activity emoji => takes you to tab?=...


## Related Issue
#56 
#57 
#44 

## Type of Change



## Testing Steps

npm run dev
do a lighthouse test after building to check performance metrics
routes to check =>
home page
feeds
